website/docs/guide/motivation.md

```diff
@@ -0,0 +1,37 @@
+---
+id: motivation
+title: Motivation
+sidebar_label: Motivation
+---
+
+[Solid](https://solidproject.org) is a wonderful project, but the technology it's built on —[RDF](https://en.wikipedia.org/wiki/Resource_Description_Framework)— is unfamiliar to many developers and has a relatively high barrier to entry.
```

Comment:
> has a relatively high barrier to entry.

No it has not, that's anti-RDF propaganda !

Reply:
We have always been at war with RDF.

Reply:
This war has been going on for far too long, and caused far too much suffering